src/tools/miri/tests/native-lib/ptr_read_access.c

```diff
@@ -1,31 +1,31 @@
 #include <stdio.h>
 
-/* Test. */
+/* Test: `test_pointer` */
```

Comment:
No need to have backtics here.
```suggestion
/* Test: test_pointer */
```